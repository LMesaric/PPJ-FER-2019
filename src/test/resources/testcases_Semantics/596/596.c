int f(int a, int b){
  return a+b;
  }
int main(void) {
 int x;
 +a=3;
 }
