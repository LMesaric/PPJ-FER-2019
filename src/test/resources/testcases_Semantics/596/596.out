<primarni_izraz> ::= IDN(6,a)
